pubsub/app.yaml

```diff
@@ -12,8 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-runtime: ruby
-env: flex
+runtime: ruby27
 entrypoint: bundle exec ruby listener.rb
 liveness_check:
```

Comment:
I think we can ditch this liveness_check for app engine standard.

Reply:
Removed, tested manually. works.